packages/flutter/lib/src/material/drawer.dart

```diff
@@ -116,19 +116,19 @@ class DrawerControllerState extends State<DrawerController> {
   }
 
   AnimationController _controller;
-  double _width = _kEdgeDragWidth;
-
-  void _handleSizeChanged(Size newSize) {
-    setState(() {
-      _width = newSize.width;
-    });
-  }
 
   void _handleTapDown(Point position) {
     _controller.stop();
     _ensureHistoryEntry();
   }
 
+  double get _width {
+    RenderBox drawerBox = _drawerKey.currentContext?.findRenderObject();
```

Comment:
There's too many entry points for "events". Touch events, keyboard events, semantics events, etc. But they all happen between frames. So I went with:
```dart
assert(!Scheduler.debugInFrame);
```